### DISCUSSION TITLE
No Protection For Unclaimed Land

### DISCUSSION THREAD

**Discussion by JJAquaBruhh** [GriefPrevention/GriefPrevention#1768] [Ideas]
I Like This Plugin.It Helped Me To Save My Base From The Destruction Of Creeper.But I Don't Want Protection In Unclaimed Land Specially in nether and the end.To Mine Ancient Debris,We Need Bed Or Tnt But We Can't Use Them Cuzz Nether Is Unclaimed.We Have To Claim The Whole Nether 😳🤯 Than Disable ExplosionsClaim 😓.Help Me 🙏🏻

**Comment by bigpresh** [comment]
See `BlockSurfaceExplosions` documented at https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration

Also, if you can't build/break blocks outside of claims, see the documentation for `claims.mode`, make sure you've not set it to `Creative` or `SurvivalRequiringClaims`
